ReSharper.FSharp/test/src/FSharp.Intentions.Tests/src/QuickFixes/UpdateMutabilityInSignatureFixTest.fs

```diff
@@ -0,0 +1,12 @@
+﻿namespace JetBrains.ReSharper.Plugins.FSharp.Tests.Intentions.QuickFixes
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Daemon.QuickFixes
+open JetBrains.ReSharper.Plugins.FSharp.Tests
+open NUnit.Framework
+
+[<FSharpTest>]
+type UpdateMutabilityInSignatureFixTest() =
+    inherit FSharpQuickFixTestBase<UpdateMutabilityInSignatureFix>()
+    override x.RelativeTestDataPath = "features/quickFixes/updateMutabilityInSignatureFix"
+    [<Test>] member x.``Add Mutability - 01`` () = x.DoNamedTestWithSignature()
+    [<Test>] member x.``Remove Mutability - 01`` () = x.DoNamedTestWithSignature()
```

Comment:
Could you please add tests covering nested patterns that were discussed above (even if keep the quick fix disabled for them)?

Reply:
Added a test for `(a:int)`. Not sure what other tests I could still add.

Reply:
Added more tests for patterns